combat-trainer.lic

```diff
@@ -1707,6 +1712,20 @@ class GameState
     @analyze_combo_type = settings.analyze_combo_type
     echo("  @analyze_combo_type: #{@analyze_combo_type}") if $debug_mode_ct
 
+    @use_barb_combos = settings.use_barb_combos
+    echo("  @use_barb_combos: #{@use_barb_combos}") if $debug_mode_ct
+
+    # For deprecation support
+    if !@analyze_combo_type.nil?
+      @use_barb_combos = true
```

Comment:
I recommend doing this before echoing the value on line 1716 (or better yet, echo something else here so we know the setting got overriden).